cli/configure/configure.go

```diff
@@ -552,6 +555,29 @@ func detectLocalTt(cliOpts *config.CliOpts) (string, error) {
 	return localCli, nil
 }
 
+// detectLocalTcm searches for available Tcm executable.
+func detectLocalTcm(cmdCtx *cmdcontext.CmdCtx, cliOpts *config.CliOpts) (string, error) {
+	localTcm, err := util.JoinAbspath(cliOpts.Env.BinDir, "tcm")
+	if err != nil {
+		return "", err
+	}
+
+	if _, err := os.Stat(localTcm); err == nil {
+		if _, err := exec.LookPath(localTcm); err != nil {
+			return "", fmt.Errorf(`found Tcm binary '%s' isn't executable: %s`,
```

Comment:
"%q" format do the quoting.
```suggestion
			return "", fmt.Errorf(`found Tcm binary %q isn't executable: %s`,
```

Reply:
Done